helm/api-testing/values.yaml

```diff
@@ -109,3 +109,15 @@ affinity: {}
 
 mongodb:
   enabled: false
+
+serviceMonitor:
+  enabled: true
```

Comment:
I prefer to change this default value to `false` because This CRD is not in the core of Kubernetes.

Reply:
I changed it to false and used.